robyn/router.py

```diff
@@ -28,7 +28,7 @@ def _format_response(self, res):
         if type(res) == dict:
             status_code = res.get("status_code", 200)
             headers = res.get("headers", {"Content-Type": "text/plain"})
-            body = res.get("body", "")
+            body = str(res.get("body", "")).encode("utf-8")
```

Comment:
After changing the Rust code to take either `bytes` or an `str`, this line can be reverted.